internal/pkg/vm/state/state.go

```diff
@@ -111,15 +111,12 @@ func (st *State) SetActor(ctx context.Context, key actorKey, a *actor.Actor) err
 }
 
 // DeleteActor remove the actor from the storage.
-//
-// This method will NOT return an error if the actor was not found.
```

Comment:
This has only one caller, which now loads the actor in question before attempting to delete it, so it's appropriate to indicate if not found.

Reply:
This was always the case, the actor being deleted is the actor that is running. We already know it exists. 

The reason deleteActor here does not leak found/not-found out is for callers not make any assumptions about it. 
Not all implementations of stores can determine if something exists before deleting, most often that not, you do not want to do the check due to performance reasons. 

Please keep the looser requirement.